### ISSUE TITLE
How to decode transaction argument 1 and 2 (recipient adress and amount)?

### ISSUE THREAD

**Issue by bobio2018** [bonustrack/libra-grpc#6]
### Expected behavior
A library to decode tx argument 1 and 2 in `rawTx.program.argumentList`

### Actual behavior

No utility function is available for decoding tx arguments

### Steps to reproduce the behavior
use get transactions and loop through each tx
```
client.request('get_transactions', params).then(function(result) {
        var rawTx = libra.utils.deserializeRawTxnBytes(i.raw_txn_bytes);
        var argument1 = rawTx.program.argumentsList[0];
	var argument2 = rawTx.program.argumentsList[1];
	console.log('argument1',argument1);
	console.log('argument2',argument2);
});
```

The arguments looks like below
```
argument1 { type: 1,
  data: 'BF8778jjnZ7UI6OOEXTwlcrubZIKthubGA426+CVJfw=' }
argument2 { type: 0, data: 'QPKACwAAAAA=' }
```

I think they are recipient address and amount, how do I decode the actual `data`?

Also is `ascending` available for get transactions query?

**Comment by giekaton** [comment]
It works like this:

**`utils.js` file**

```
import { RawTransaction } from './pb/transaction_pb';


function decodeRawTx(rawTxnBytes) {
  const rawTxn = RawTransaction.deserializeBinary(rawTxnBytes);
  const rawProgram = rawTxn.getProgram();

  const program = {
    arguments: rawProgram.getArgumentsList().map(argument => ({
      type: argument.getType(),
      value: argument.getData_asU8(),
    }))
  };

  let address = Buffer.from(program.arguments[0].value, 'base64').toString('hex');

  let value = parseInt(bufferToHex(program.arguments[1].value), 16);

  function bufferToHex (buffer) {
    return Array
      .from(new Uint8Array(buffer))
      .reverse()
      .map(b => b.toString(16).padStart(2, '0'))
      .join('')
  }

  return( { address, value } );
}


export default {
  decodeRawTx
}
```

**`test.js` file**

```
const libra = require('..');

const client = new libra.Client('ac.testnet.libra.org:8000');


const test = async () => {
  
  /** Get transactions */
  params = {
    start_version: 142864,
    limit: 1,
    fetch_events: false,
  };

  client.request('get_transactions', params).then(result => {

    result.txn_list_with_proof.transactions.forEach(tx => {

      let decoded = libra.utils.decodeRawTx(tx.raw_txn_bytes);
      console.log(decoded);
      
    });

  });

}

test();
```

**Comment by bonustrack** [comment]
@bobio2018 @giekaton We just released libra-grpc@0.0.5, this version decode the result of "get_transactions" directly using @giekaton snippet.
